apps/main/src/loan/store/createScrvUsdSlice.ts

```diff
@@ -418,7 +359,11 @@ const createScrvUsdSlice = (set: SetState<State>, get: GetState<State>) => ({
             errorMessage: '',
           })
           dismissNotificationHandler()
-          get()[sliceKey].fetchUserBalances()
+
+          // invalidate user balances query
+          const signerAddress = useWallet.getState().wallet?.accounts?.[0]?.address.toLowerCase()
+          queryClient.invalidateQueries({ queryKey: ['scrvUsdUserBalances', { signerAddress: signerAddress ?? '' }] })
```

Comment:
The queryFactory returns an `invalidate` function. Also, it exports a `queryKey` function so we never need to hardcode query keys (we might want to change them). Finally, `getQueryData` is also exported. Example:

https://github.com/curvefi/curve-frontend/blob/d994ebfadda16886c590b5a2a8a7cfe7ad086653/apps/main/src/loan/entities/lending-vaults.ts#L94